examples/nlp/text_classification_tf_keras/README.md

```diff
@@ -0,0 +1,37 @@
+# Tensorflow Multi-class Text Classification Example
+
+This example demonstrates how to integrate and train a Tensorflow model on Determined's API. 
+We will implement a multiclass classifier that will be trained to perform sentiment 
+analysis on a dataset of Amazon product reviews, classifying reviews into 5 categories (0-5 stars). 
+It is adapted from Tensorflow's
+[word embedding](https://www.tensorflow.org/tutorials/text/word_embeddings) tutorial.
+
+## Files
+* `model_def.py`: main trial class for your model containing the model definition, initialization
+  methods, and methods for loading training and validation data.
+  
+* `data.py`: methods for downloading and processing data and datasets for use in the model
+
+###Configuration Files
+* `const.yaml`: model training with constant hyperparameter values
+* `distribyted.yaml`: trains the model with multiple GPUs (distributed training)
+* `adaptive.yaml`: performs hyperparameter search on specified hyperparameter values
```

Comment:
adaptive.yaml: Perform a hyperparameter search using Determined's state-of-the-art adaptive hyperparameter tuning algorithm.

Reply:
done